lib/Sema/CSSimplify.cpp

```diff
@@ -2958,6 +2958,19 @@ repairViaOptionalUnwrap(ConstraintSystem &cs, Type fromType, Type toType,
   std::tie(fromObjectType, fromUnwraps) = getObjectTypeAndUnwraps(fromType);
   std::tie(toObjectType, toUnwraps) = getObjectTypeAndUnwraps(toType);
 
+  // Since equality is symmetric and it decays into a `Bind` eagerly
```

Comment:
I suppose there's a missing comma here before `eagerly`.

Reply:
Fixed, thank you!